test/lib/next-modes/next-deploy.ts

```diff
@@ -30,7 +30,7 @@ export class NextDeployInstance extends NextInstance {
       require('console').log(`Using Vercel CLI version:`, res.stdout)
     } catch (_) {
       require('console').log(`Installing Vercel CLI`)
-      await execa('npm', ['i', '-g', 'vercel@latest'], {
+      await execa('npm', ['i', '-g', 'vercel@28.13.2'], {
```

Comment:
This is only triggered for a release currently, can change this to run here https://github.com/vercel/next.js/blob/1b750e18b1e5b0668857fc1d5a023d0578122fca/.github/workflows/build_test_deploy.yml#L799